Directory.Build.props

```diff
@@ -9,6 +9,7 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)Renci.SshNet.snk</AssemblyOriginatorKeyFile>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <EnablePackageValidation>true</EnablePackageValidation>
```

Comment:
unrelated change, this just adds validation during `dotnet pack` that e.g. a netstandard build is compatible with netcore

https://learn.microsoft.com/dotnet/fundamentals/apicompat/package-validation/overview